src/components/map/components/legend/legend-single-features-layer-with-single-style.stories.js

```diff
@@ -0,0 +1,63 @@
+import { html } from 'lit-html';
+import '../../index.js';
+import { argTypes } from './config/index.js';
+import { docsIntro } from '../../../../../.storybook/utils.js';
+import { LEGEND_PLACEMENT } from './enums/index.js';
+
+export default {
+  title: 'custom-elements/vl-map/vl-map-legend/vl-map-legend-single-features-layer-with-single-style',
+  parameters: {
+    controls: { hideNoControlsWarning: true },
+    docs: {
+      description: {
+        component: docsIntro({
+          root: 'map',
+          intro: 'De kaart legende voor kaart met één vl-map-features-layer met één vl-map-layer-style.',
+        }),
+      },
+    },
+  },
+  argTypes: {
+    ...argTypes,
+  },
+};
+
+export const Default = () => {
+  const features = {
+    type: 'FeatureCollection',
+    features: [
+      {
+        type: 'Feature',
+        geometry: {
+          type: 'Point',
+          coordinates: [147055.0, 197908.0],
+        },
+      },
+      {
+        type: 'Feature',
+        geometry: {
+          type: 'Point',
+          coordinates: [141000.0, 200908.0],
+        },
+      },
+    ],
+  };
+
+  return html`<vl-map id="map">
+    <vl-map-baselayer-grb-gray></vl-map-baselayer-grb-gray>
+    <vl-map-features-layer .features=${features} data-vl-name="Laag 1">
+      <vl-map-layer-circle-style
+        data-vl-legend-text="Openbaar onderzoek"
+        data-vl-color="#ffe615"
+        data-vl-size="5"
+        data-vl-border-color="#000"
+        data-vl-border-size="1"
+      ></vl-map-layer-circle-style>
```

Comment:
Dit, alsook het voorbeeld met "meerdere lagen met elk maar 1 style" is in principe wat standaard meest gebruikt wordt. 
In dat geval, zou ik de data-vl-name van elke laag gebruiken en die renderen, ipv nog eens verplicht te worden om dat nog eens mee te geven bij je style. 
Het is pas als je met meerdere styles voor 1 laag zit (wat meestal ook wel opgelost kan worden dus door verschillende lagen te gebruiken) dat je moet gaan differentieren.
Ik zou de component dus slimmer maken, in die zin dat hij bij 1 layer - style de naam van de laag gebruikt, en slechts bij meerdere styles de "legend-text" van de style, waarbij ik ook eventueel de naam van het attribuut eerder "name" zou noemen ipv "legend-text"

Reply:
De "data-vl-name" van je laag wordt bvb ook gebruikt bij de layer-switcher, en je wil liever niet dat die uit mekaar beginnen te lopen.

Reply:
aanpassingen aangebracht zoals omschreven door Pascal